packages/vue-components/src/questions/Quiz.vue

```diff
@@ -128,12 +128,13 @@ export default {
   },
   methods: {
     gotoNextQuestion() {
+      const DELAY = 525;
```

Comment:
Might be better to define this outside the method since I see it also used inline 153 :)

Reply:
Yep I agree! I was trying to do that under methods but only functions can be declared in methods. I also tried implementing a getDelay() function but eslint threw me an error so I decided on this instead! Maybe I can try putting it under a different hook. What do you think?

Reply:
I thinking more along the lines of pulling out the constant DELAY with other constants like STATE_QUIZ_FRESH in line 96 so other methods can reference this as well. What do you think?

Reply:
Ahh I see. I think thats a great idea! I will update that in the next commit!